File: backend/src/main/java/project/backend/global/redis/dao/TokenRedisRepository.java
package project.backend.global.redis.dao;

import java.util.Optional;

import org.springframework.data.repository.CrudRepository;
import project.backend.global.redis.entity.TokenRedis;

public interface TokenRedisRepository extends CrudRepository<TokenRedis, Long> {

	Optional<TokenRedis> findByAccessToken(String accessToken);
}

File: backend/src/main/java/project/backend/auth/token/dao/TokenRedisRepository.kt
package project.backend.auth.token.dao

import org.springframework.data.repository.CrudRepository
import project.backend.auth.token.entity.TokenRedis
import java.util.Optional

interface TokenRedisRepository : CrudRepository<TokenRedis, Long> {
    fun findByAccessToken(accessToken: String): Optional<TokenRedis>
}